strum_macros/Cargo.toml

```diff
@@ -24,6 +24,7 @@ proc-macro2 = "1.0"
 quote = "1.0"
 rustversion = "1.0"
 syn = { version = "1.0", features = ["parsing", "extra-traits"] }
+convert_case = "0.6"
```

Comment:
convert_case has a dev_dependency on `strum`. Can we use `heck` to avoid pulling multiple casing libs?

Reply:
Seems like heck is handling numbers differently from convert-case, e.g.
heck:               "Number1Function" => "number1_function".
convert-case : "Number1Function" => "number_1_function".

Personally I prefer that numbers are treated as separate words like conver-case does, but can change it if you disagree.

See withoutboats/heck/issues/18

Reply:
I agree that heck's behavior here is a little odd, but I'd prefer to not introduce a new dependency. Let's either, just do what heck does, or take heck's output and just insert a `_` before a number if it's preceded by a letter.

Reply:
Removed convert_case